api/v2/serializers/details/resource_request.py

```diff
@@ -194,6 +194,13 @@ class Meta:
         )
 
 
+def _get_pending_status():
+    try:
+        return StatusType.objects.get(name="pending")
+    except Exception:
```

Comment:
Lets do an 'except DoesNotExist as e' so we don't catchall.
1. As a good habit
2. MultipleObjectsReturned would notify us that our db is messed up